page_creation.rst

```diff
@@ -18,10 +18,11 @@ two-step process:
    create a Symfony ``Response`` object, which can hold HTML content, a JSON
    string or even a binary file like an image or PDF.
 
-.. seealso::
+.. admonition:: Screencast
+    :class: screencast
 
     Do you prefer video tutorials? Check out the `Joyful Development with Symfony`_
-    screencast series from KnpUniversity.
+    screencast series.
```

Comment:
We should update the URL of this one too :) https://knpuniversity.com/screencast/symfony3